.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: "\U0001F41B Bug report"
+about: Submit a bug report to help us improve LLM Engine. If this doesn't look right, [choose a different type.](https://github.com/scaleapi/llm-engine/issues/new/choose)
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+Thank you for taking the time to file a bug report! Before you do so, please take a look at existing open issues and make sure that your issue is not already documented. If it isn't, please provide us with a clear and concise description of what the bug is.
+
+**LLM Engine Version**
+- LLM Engine Version: 
+
+**System Version**
+- Python Version: 
+- Operating System: 
+
+**Timestamp and Request ID**
+_If you ran into an internal error while using `llm-engine`, please provide the following_
+- Timestamp:
```

Comment:
Maybe write `timestamp` and `request_id` to highlight that these are literal fields from the response? And also explicitly mention that they're from the response. Just to avoid any situations where someone reads `Timestamp` and thinks of it as an estimate from their memory.